src/System.Private.Xml/tests/XmlSerializer/XmlSerializerTests.cs

```diff
@@ -1659,6 +1659,16 @@ public static void Xml_TypeWithTimeSpanProperty()
     }
 
     [Fact]
+    public static void Xml_TypeWithDefaultTimeSpanProperty()
+    {
+        var obj = new TypeWithDefaultTimeSpanProperty { TimeSpanProperty = new TimeSpan(0, 1, 0) };
```

Comment:
Is there a test anywhere which verifies that a non-default value roundtrips successfully?